.travis.yml

```diff
@@ -30,8 +30,7 @@ jdk:
   - openjdk11
 branches:
   only:
```

Comment:
This shouldn't be changed in .travis.yml. Please, rollback the change.

Reply:
Fixed in 4d08affe85545b26afed5461585bc8bd710d5a85